app/controllers/application_controller.rb

```diff
@@ -12,12 +14,46 @@ class ApplicationController < ActionController::Base
 
   # Determine if the collection is viewable depending on private flag and white-listed remote IP address
   def is_viewable? (collection)
-    !collection.is_private || (collection.allowed_ip_addresses.split(%r{,\s*}).include? request.remote_ip)
+
+    # If no ip address filtering
+    if (collection['allowed_ip_addresses'].empty?)
+      if (is_private?(collection))
+        # TODO: Brittle - Allow better control over who may view a private collecionts
+        RoleMapper.roles(current_user.email).include? "archivist"
```

Comment:
@skng5 I need a little help understanding this line. Why are we checking if the email of the current user contains the string "archivist"? Is that a default email address of some kind for admin users?